backend/src/main/java/org/eclipse/tractusx/puris/backend/common/util/VariablesService.java

```diff
@@ -56,6 +56,14 @@ public class VariablesService {
      */
     private String edrEndpoint;
 
+
+    /**
+     * The endpoint of your ERP adapter
+     * (currently just mock!)
+     */
+//    @Value("${puris.erpadapterurl")
+    private String erpAdapterUrl ="http:foo.com/my_erp_endpoint";
```

Comment:
Ah now I see that we should add this url to the Helm Chart! Version increase should be major. Currently we don't yet use it, don't we?